components/zcash_address/src/kind/unified/ivk.rs

```diff
@@ -99,51 +95,75 @@ impl SealedItem for Ivk {
 /// // We can obtain the pool-specific Incoming Viewing Keys for the UIVK in
 /// // preference order (the order in which wallets should prefer to use their
 /// // corresponding address receivers):
```

Comment:
This comment is no longer correct for `uivk.items_as_parsed()`.

Reply:
:heavy_check_mark: